modules/claims_api/app/swagger/claims_api/v1/form_526_controller_swagger.rb

```diff
@@ -140,7 +140,11 @@ class Form526ControllerSwagger
             key :description, '526 response'
             content 'application/json' do
               schema do
-                key :'$ref', :ClaimsIndex
+                key :type, :object
```

Comment:
@bastosmichael before I go too deep on updating these, I wanna make sure I'm not adding `data` properties to the top level of any responses that don't need it. It was unclear looking through the JSON API spec if that formatting applies to `POST`/`PUT` requests in addition to `GET`

Reply:
yes the data element applies to both the POST and PUT as well as the GET. JSON API spec treats the whole thing as a data model. The reason we return the ClaimsIndex instead of a 526 specific payload is because of the PII and PHI nature of the 526 payload itself and ClaimsIndex is what they get back when they check for status anyhow.